.husky/pre-commit

```diff
@@ -1,5 +0,0 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
```

Comment:
Is this supposed to be removed?

Reply:
Nope. Good spot. Not sure how that happened. 😅